local-interchain/cmd/local-ic/root.go

```diff
@@ -9,10 +9,6 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var (
-	MakeFileInstallDirectory string
-)
```

Comment:
Avoid global vars. In this case, wasn't needed at all.

Reply:
This is required for the ldflags in `makefile` to work? or does the var in get directory work properly when you `make install`. This is the reason it is a globalvar